app/src/main/groovy/ua/net/nlp/api/BatchController.groovy

```diff
@@ -57,13 +75,18 @@ class BatchController extends BaseController {
 
     @Schema(description="Request to clean, tokenize, and lemmatize text.")
     @CompileStatic
-    static class BatchRequest extends RequestBase {
+    static class BatchRequest {
+        @Schema(description="Texts to process.")
+        List<String> texts
     }
 
+
     @Schema(description="Response with tokenized and lemmatized text.")
     @CompileStatic
     static class BatchResponse {
+        String clean
         List<List<String>> tokenized
         List<List<String>> lemmatized
+        List<String> segmented
```

Comment:
I wonder if should call this 'sentences' (and the other two 'tokens' and 'lemmas'?